### DISCUSSION TITLE
Error: x509: certificate signed by unknown authority while apply the sample yaml

### DISCUSSION THREAD

**Discussion by zhixunjie** [kubernetes-sigs/kubebuilder#2685] [Support]
➜ demo-operator git:(master) ✗ k apply -f config/samples/apps_v1beta1_mydaemonset.yaml
Error from server (InternalError): error when creating "config/samples/apps_v1beta1_mydaemonset.yaml": Internal error occurred: failed calling webhook "mmydaemonset.kb.io": failed to call webhook: Post "[https://demo-operator-webhook-service.demo-operator-system.svc:443/mutate-apps-cncamp-io-v1beta1-mydaemonset?timeout=10s](https://demo-operator-webhook-service.demo-operator-system.svc/mutate-apps-cncamp-io-v1beta1-mydaemonset?timeout=10s)": x509: certificate signed by unknown authority

the certificate is ok!
root@k8s-master-a-1:tt# k get certificate -n demo-operator-system
NAME READY SECRET AGE
demo-operator-serving-cert True webhook-server-cert 81m

controller manager's log here：
2022/05/13 17:50:33 http: TLS handshake error from 192.168.34.2:43156: remote error: tls: bad certificate

**Comment by camilamacedo86** [comment]
@zhixunjie, 

Did you enable the cert-manager?
Did you check that the secret is available like we do in the e2e tests, see [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v3/plugin_cluster_test.go#L264-L270)?

Could you please provide the steps over how can we create a project and face this issue?

**Comment by camilamacedo86** [comment]
closing since no extra info was provided for we see how we could help
feel free to raise issues if you need as well
